test/views/inp.njk

```diff
@@ -118,7 +118,7 @@
       inp.instance = 1;
 
       // Log for easier manual testing.
-      console.log(inp);
+      console.log('INP1', inp);
```

Comment:
Did you want to do this everywhere, or just for INP?

Reply:
Good point. Fixed em all.